--- origsrc/audacious-3.4.3/acinclude.m4	2014-01-03 18:53:09.000000000 -0600
+++ src/audacious-3.4.3/acinclude.m4	2014-01-31 04:24:18.471216300 -0600
@@ -186,10 +186,10 @@ AC_PROG_MAKE_SET
 AC_PATH_PROG([RM], [rm])
 AC_PATH_PROG([MV], [mv])
 AC_PATH_PROG([CP], [cp])
-AC_PATH_PROG([AR], [ar])
+AC_PATH_TOOL([AR], [ar])
 AC_PATH_PROG([TR], [tr])
-AC_PATH_PROG([RANLIB], [ranlib])
-AC_PATH_PROG([WINDRES], [windres])
+AC_PATH_TOOL([RANLIB], [ranlib])
+AC_PATH_TOOL([WINDRES], [windres])
 
 dnl Check for POSIX threads
 dnl =======================
