--- origsrc/audacious-3.6.2/acinclude.m4	2015-05-30 17:27:01.000000000 -0500
+++ src/audacious-3.6.2/acinclude.m4	2015-07-17 16:17:43.230297500 -0500
@@ -110,12 +110,6 @@ if test $HAVE_LINUX = yes ; then
 	LDFLAGS="$LDFLAGS -Wl,-z,defs"
 fi
 
-dnl MinGW needs -march=i686 for atomics
-dnl ===================================
-if test $HAVE_MSWINDOWS = yes ; then
-	CFLAGS="$CFLAGS -march=i686"
-fi
-
 dnl Byte order
 dnl ==========
 AC_C_BIGENDIAN([BIGENDIAN=1], [BIGENDIAN=0],
@@ -142,9 +136,9 @@ AC_PROG_LN_S
 AC_PATH_PROG([RM], [rm])
 AC_PATH_PROG([MV], [mv])
 AC_PATH_PROG([CP], [cp])
-AC_PATH_PROG([AR], [ar])
-AC_PATH_PROG([RANLIB], [ranlib])
-AC_PATH_PROG([WINDRES], [windres])
+AC_PATH_TOOL([AR], [ar])
+AC_PATH_TOOL([RANLIB], [ranlib])
+AC_PATH_TOOL([WINDRES], [windres])
 
 dnl Check for POSIX threads
 dnl =======================
--- origsrc/audacious-3.6.2/src/libaudcore/runtime.cc	2015-05-30 17:27:01.000000000 -0500
+++ src/audacious-3.6.2/src/libaudcore/runtime.cc	2015-07-17 16:21:01.537979400 -0500
@@ -95,7 +95,7 @@ EXPORT MainloopType aud_get_mainloop_typ
 
 static StringBuf get_path_to_self ()
 {
-#ifdef HAVE_PROC_SELF_EXE
+#if defined(__linux__) || defined(__CYGWIN__)
 
     StringBuf buf (-1);
     int len = readlink ("/proc/self/exe", buf, buf.len ());
